tuning/sft_trainer.py

```diff
@@ -418,8 +410,9 @@ def train(
         )
 
     trainer.train(resume_from_checkpoint)
-
-    return trainer
+    additional_metadata = {}
+    additional_metadata["added_tokens_info"] = added_tokens_dict
+    return trainer, additional_metadata
```

Comment:
Returning additional metadata from train() containing information of newly added tokens. @kmehant let me know if this change is ok with you - it might be disruptive if using SDK and if you were relying on 1 return value. We can make this a major release with the change.

Alternatively, I can write the file artifacts I want inside 'output_dir' and later can copy it to save_model_dir as well, if return value is an issue. I just thought this return value could be extendable to any other metadata we need in future

Reply:
If we remove custom addition of tokens from fms-hf-tuning, we might not need to return this added tokens metadata.